swap/tests/bob_refunds_using_cancel_and_refund_command_timelock_not_expired.rs

```diff
@@ -10,6 +10,7 @@ use swap::protocol::{alice, bob};
 async fn given_bob_manually_cancels_when_timelock_not_expired_errors() {
     harness::setup_test(SlowCancelConfig, |mut ctx| async move {
         let (bob_swap, bob_join_handle) = ctx.bob_swap().await;
+        let bob_swap_id = bob_swap.swap_id;
```

Comment:
Given that the swap id is the same for both, I think we don't need to qualify with a role here.